packages/lesswrong/server/resolvers/tagResolvers.ts

```diff
@@ -379,11 +379,11 @@ async function buildContributorsList(tag: DbTag, version: string|null): Promise<
       const selfVoteAdjustment = selfVoteScoreAdjustmentByUser[userId]
       const excludedPower = selfVoteAdjustment?.excludedPower || 0;
       const excludedVoteCount = selfVoteAdjustment?.excludedVoteCount || 0;
-      
+
       return {
         contributionScore: totalRevisionScore - excludedPower,
         numCommits: revisionsByThisUser.length,
-        voteCount: sumBy(revisionsByThisUser, r=>r.voteCount) - excludedVoteCount,
+        voteCount: sumBy(revisionsByThisUser, r=>r.voteCount ?? 0) - excludedVoteCount,
```

Comment:
I hit a bug where `voteCount` could be NaN

Reply:
lol - I feel like this indicates an issue somewhere else (unless it's just old crufty data?)